logfile.go

```diff
@@ -20,26 +22,44 @@ func lineWorker(die dieCh, r *bufio.Reader, cfg logplexc.Config, sr *serveRecord
 		log.Fatalf("could not create logging client: %v", err)
 	}
 
-	for {
-		select {
-		case <-die:
-			return
-		default:
-			break
-		}
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Why do you want to kill the program in this fatal? A few words for "blow up the entire process" seems worthwhile (also, have fun debugging without it).